ui/src/pages/frontsite/components/Series/SeriesLiteratureSearch/SeriesLiteratureSearch.js

```diff
@@ -63,7 +63,11 @@ export class SeriesLiteratureSearch extends React.Component {
         <Divider horizontal>
           Literatures in this {metadata.mode_of_issuance.toUpperCase()}
         </Divider>
-        <ReactSearchKit searchApi={api} history={history}>
+        <ReactSearchKit
+          searchApi={api}
+          history={history}
+          urlHandlerApi={{ enabled: false }}
```

Comment:
This removes url deep linking in series details page, not needed when rendering ReactSearchKit with the list of documents attached to a series.